pop-api/src/v0/fungibles/traits.rs

```diff
@@ -113,9 +121,12 @@ pub trait Psp22Mintable {
 pub trait Psp22Burnable {
 	/// Destroys `value` amount of tokens from `account`, reducing the total supply.
 	///
+	/// The selector for this message is `0xfc3c75d4` (first 4 bytes of
+	/// `blake2b_256("PSP22Mintable::mint")`).
+	///
 	/// # Parameters
 	/// - `account` - The account from which the tokens will be destroyed.
 	/// - `value` - The number of tokens to destroy.
-	#[ink(message)]
+	#[ink(message, selector = 0x7a9da510)]
```

Comment:
This doesnt match the comment, think you have them switched with mint.

Reply:
The comment was wrong. Resolved